roles/mariadb/templates/my.cnf.j2

```diff
@@ -0,0 +1,3 @@
+[client]
+user=root
```

Comment:
We could do that, but it wouldn't work if the user changer the value afterward, because MySQL save the users in the database, we would need to implement a way to modify the root user via a MySQL command. I looked around, and this is not recommended. (Places like Amazon RDS that lets you set the root user won't let you change afterwards. So the `mysql_root_user` is only used on external installs. )